torch/distributed/_tools/memory_tracker.py

```diff
@@ -147,36 +143,86 @@ def summary(self, top: int = 20) -> None:
         print("------------------------------------------------")
 
     @no_type_check
-    def show_traces(self) -> None:
+    def show_traces(self, path: str = "") -> None:
```

Comment:
Sorry, maybe these PRs are not ready for review. This diff to `show_traces()` looks different than the one in https://github.com/pytorch/pytorch/pull/90144.